src/main/java/stream/StreamConfiguration.java

```diff
@@ -19,6 +19,7 @@ public class StreamConfiguration extends Properties {
     public StreamConfiguration() {
         super();
         put(StreamsConfig.APPLICATION_ID_CONFIG, Config.GROUP_ID);
+        put(StreamsConfig.COMMIT_INTERVAL_MS_CONFIG, 1000);
```

Comment:
I suggest we make this configurable, we might see some cases where we need this to be higher or less. especially this implementation being a long running aggregation

Reply:
done